ui/app/components/app/signature-request/signature-request-header/signature-request-header.component.js

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import AccountListItem from '../../../../pages/send/account-list-item/account-list-item.component'
+import NetworkDisplay from '../../network-display'
+
+export default class SignatureRequestHeader extends PureComponent {
+  static propTypes = {
+    selectedAccount: PropTypes.object.isRequired,
+    accounts: PropTypes.object,
+  }
+
+  render () {
+    const { selectedAccount } = this.props
+    console.log('selectedAccount', selectedAccount)
```

Comment:
Nit: console.log

Reply:
Fixed in c8120c6dac920b8a64d46fce781d099aa398c8b1